torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -3271,11 +3273,26 @@ def parse_env(var):
             self.assertEqual(ddp_logging_data.nccl_debug, parse_env("NCCL_DEBUG"))
             self.assertEqual(ddp_logging_data.nccl_nthreads, parse_env("NCCL_NTHREADS"))
             self.assertEqual(ddp_logging_data.nccl_ib_timeout, parse_env("NCCL_IB_TIMEOUT"))
+            # test runtime logging fields
+            self.assertEqual(ddp_logging_data.unused_parameter_size, 0)
```

Comment:
Can we add a test where there are unused paramters and verify this value (can be done in a follow up diff)?